### ISSUE TITLE
ls with symbol link get failed

### ISSUE THREAD

**Issue by iysheng** [Peltoche/lsd#463]
When I am in a symbolic linked directory, I want to ls with the parent of this directory as **ls ../**,It's ok as below, but when i want to see a directory of parent, it's get failed. the print as blow:
![ZtX3m6.png](https://t1.picb.cc/uploads/2020/12/16/ZtX3m6.png)

**Comment by meain** [comment]
I was not able to reproduce this in latest version. Could you try with the latest version `0.19.0`.

**Comment by sklages** [comment]
I cannot reproduce neither (0.19.0):

```
[user_abc] ~/temp/x
16.12.2020 @ 11:48:14  ❯ ls -l
drwxrwxr-x user_abc user_abc 4.0 KB Wed Dec 16 11:48:09 2020  data
lrwxrwxrwx user_abc user_abc   4 B  Wed Dec 16 11:44:39 2020  data_link ⇒ data

[user_abc] ~/temp/x
16.12.2020 @ 11:48:19  ❯ cd data_link/

[user_abc] ~/temp/x/data_link
16.12.2020 @ 11:48:28  ❯ ls -l ../
drwxrwxr-x user_abc user_abc 4.0 KB Wed Dec 16 11:48:09 2020  data
lrwxrwxrwx user_abc user_abc   4 B  Wed Dec 16 11:44:39 2020  data_link ⇒ data

[user_abc] ~/temp/x/data_link
16.12.2020 @ 11:48:36  ❯ ls -l ../data
.rw-rw-r-- user_abc user_abc 0 B Wed Dec 16 11:48:09 2020  xxx

[user_abc] ~/temp/x/data_link
16.12.2020 @ 11:48:40  ❯

```

**Comment by zwpaper** [comment]
There is a bug in version 0.16.0, and should be fixed after 0.17.0

https://github.com/Peltoche/lsd/issues/72

**Comment by iysheng** [comment]
yeah, after i upgrade lsd from 0.17 to 0.18, it is ok now.
